client/js/components/procedure/publicindex/DpSearchProcedureMap.vue

```diff
@@ -12,39 +12,39 @@
     <div :class="prefixClass('c-proceduresearch__search-wrapper layout__item flex')">
       <dp-autocomplete
         v-if="dplan.settings.useOpenGeoDb"
-        data-cy="searchProcedureMapForm:procedureSearch"
-        name="search"
         id="procedure_search"
-        :class="prefixClass('c-proceduresearch__search-field')"
         ref="autocomplete"
         v-model="currentAutocompleteSearch"
+        :class="prefixClass('c-proceduresearch__search-field')"
+        data-cy="searchProcedureMapForm:procedureSearch"
+        label="value"
+        name="search"
+        :height="'34px'"
```

Comment:
You can simplify it like:
```suggestion
        height="34px"
```
can we somehow "unmagic" that Number?

Reply:
> can we somehow "unmagic" that Number?

I had no good idea how. The height prop in DpAutocomplete is used for the `box-height` in `vue-omnibox`. 
Order done in [8c56701](https://github.com/demos-europe/demosplan-core/pull/3235/commits/8c56701cd9d111abc8ed4b25ef2292cff8b248ad)